ror/spec/factories/articles.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+
+  factory :article do
+    title  { 'factory article title' }
```

Comment:
Since title must be unique, this will break when you create many of them in tests.

```suggestion
    title  { |counter| "factory article title #{counter}" }
```

Reply:
Fixed.